src/sse_events.rs

```diff
@@ -0,0 +1,154 @@
+use crate::{
+    console::{ConsoleOutput, SerializableEvent},
+    demo::{DemoEvent, DemoMessage, VoteCastEventWrapped, VoteRelatedEvent},
+    state::MACState,
+    web::broadcast_event,
+};
+use event_loop::{try_get, Handled, HandlerStruct, Is};
+use std::collections::HashMap;
+
+/// This struct will house the relevant vars used by various message handlers when broadcasting
+/// Encapsulates a set of functions to invoke the `web::broadcast_event` function with the
+/// appropriately serialised JSON messages.
+pub struct BroadcastableEvent {
```

Comment:
The name `BroadcastableEvent` makes it sound like this struct is the event itself, I liked the naming scheme you had before like `SseBroadcaster` or something along those lines.

Reply:
Valid, i had double assigned the `Event` suffix in my mind to two different meanings. Went for `SseEventBroadcaster`.